### PR TITLE
native: Add SDL3 WSI (for real)

### PR DESCRIPTION
**Pull request by doitsujin**
See #4326 , #4323 

Needs more testing, but it builds and I managed to create a window with a swap chain.

CC @flibitijibibo

### CODE REVIEW

**Comment by flibitijibibo** [comment]
Will try to test this weekend - if you want a full game to try, FEZ has an 'sdl3' Steam beta, and copying the dxvk libraries to the game's lib64 folder and passing /gldevice:D3D11 as a launch option should run with dxvk over SDL3.

File: src/wsi/sdl3/wsi_platform_sdl3.cpp [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+#include <SDL3/SDL_vulkan.h>
+
+namespace dxvk::wsi {
+
+  Sdl3WsiDriver::Sdl3WsiDriver() {
+    libsdl = LoadLibraryA( // FIXME: Get soname as string from meson
+#if defined(_WIN32)
+        "SDL3.dll"
+#elif defined(__APPLE__)
+        "libSDL3.dylib"
```
Comment (flibitijibibo) [inline src/wsi/sdl3/wsi_platform_sdl3.cpp:17]:
```suggestion
        "libSDL3.0.dylib"
```

**Review by flibitijibibo — APPROVED** [review]
Just one nit, otherwise lgtm - tested with FEZ and TMNT Shredder's Revenge.

File: src/wsi/sdl3/wsi_platform_sdl3.cpp (hunk 1 above)
Comment (doitsujin) [reply src/wsi/sdl3/wsi_platform_sdl3.cpp]: Thanks, wasn't sure what the Apple library was called.

**Pull request merged**